### PR TITLE
Ocutils class replacements

### PR DESCRIPTION
**Pull request by evsiretanu**

### CODE REVIEW

File: py/py3/ocutils.py [hunk 1]
```diff
@@ -313,15 +309,15 @@ def process_level(self, level, old_models, clear_cache_flag):
                 len(best_models) < self._search_width
             ):  # or key[0] == last_key[0]:      # comparing keys allows us to select more than <width> models,
                 if True not in [
-                    n.isEquivalentTo(candidate) for n in best_models
+                    n.is_is_equivalent_to(candidate) for n in best_models
```
Comment (IanMcBride1) [inline py/py3/ocutils.py]: is this name a bit redundant? is_is_equivalent_to()

File: py/occampy3/wrappers/model.py [hunk 2]
```diff
@@ -55,3 +63,12 @@ def set_id(self, id_: int) -> None:
 
     def set_progenitor(self, progenitor: 'Model') -> None:
         self._ref.setProgenitor(progenitor.ref)
+
+    def get_attribute_value(self, attribute: str) -> Union[str, float]:
+        return self._ref.get(attribute)
+
+    def is_is_equivalent_to(self, model: 'Model') -> bool:
```
Comment (tim-coutinho) [inline py/occampy3/wrappers/model.py]: Could define `__eq__` in this way as well, to enable comparing models using `==`.

File: py/occampy3/wrappers/model.py [hunk 3]
```diff
@@ -19,10 +20,17 @@ def __init__(self, ref=None) -> None:
         self._ref = ref
         self._id = 0
 
+    def __lt__(self, other: 'Model') -> bool:
+        return self.name < other.name
```
Comment (tim-coutinho) [inline py/occampy3/wrappers/model.py]: I don't see `Model` having a `name` field anywhere.

File: py/py3/basic.py [hunk 4]
```diff
@@ -9,7 +9,10 @@
 import sys
 import time
 
+sys.path.insert(0, "./wrappers")
+
 from ocutils import OCUtils
+from wrappers.report import SortDirection
```
Comment (tim-coutinho) [inline py/py3/basic.py:15]: I think ideally we should find a way to work around working with anything in wrappers directly, outside of occampy. Some way to eventually abstract this functionality outside of these files would be good.

File: py/occampy3/wrappers/model.py (hunk 2 above)
Comment (evsiretanu) [reply py/occampy3/wrappers/model.py]: Not sure if in this case equivalence would be the same as equality

File: py/py3/basic.py (hunk 4 above)
Comment (evsiretanu) [reply py/py3/basic.py:15]: It will be that way when we create the other classes

File: py/occampy3/wrappers/model.py (hunk 2 above)
Comment (tim-coutinho) [reply py/occampy3/wrappers/model.py]: I think it works here, as `==` implies functional equivalence while `is` implies being literally the same.

Comment (evsiretanu) [reply py/occampy3/wrappers/model.py]: In my understanding `==` would mean that both `model` objects are the same, meaning that they have the same name, values, etc.. But yes, I looked in the C++ code and it does actually compare them with the `==` operator. Created a new commit with that changed

File: py/py3/ocutils.py (hunk 1 above)
Comment (evsiretanu) [reply py/py3/ocutils.py]: This method has been removed in the new commit

**Review by tim-coutinho — APPROVED** [review]

**Pull request merged**